docs/notebooks/getting_started/particles.ipynb

```diff
@@ -0,0 +1,903 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6e8f0d92",
+   "metadata": {},
+   "source": [
+    "# Using PlasmaPy Particles"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c7eeced",
+   "metadata": {},
+   "source": [
+    "[plasmapy.particles]: ../../particles/index.rst\n",
+    "\n",
+    "The [plasmapy.particles] subpackage contains functions to access basic particle data and classes to represent particles."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24125310",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from plasmapy.particles import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "650d57e8",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "\n",
+    "1. [Particle properties](#Particle-properties)\n",
+    "2. [Particle objects](#Particle-objects)\n",
+    "3. [Custom particles](#Custom-particles)\n",
+    "4. [Molecules](#Molecules)\n",
+    "5. [Particle lists](#Particle-lists)\n",
+    "6. [Dimensionless particles](#Dimensionless-particles)\n",
+    "7. [Nuclear reactions](#Nuclear-reactions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e224284a",
+   "metadata": {},
+   "source": [
+    "## Particle properties"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7696deba",
+   "metadata": {},
+   "source": [
+    "There are several functions that provide information about different particles that might be present in a plasma. The input of these functions is a representation of a particle, such as a string for the atomic symbol or the element name."
```

Comment:
Does the convention for these strings come from some standard? If so, I would add a link to that standard. If not, maybe make clear that this is a PlasmaPy-specific standard.

Reply:
Good point.  This is a PlasmaPy-specific thing, and I should add a link to ``` :term:`particle-like` ``` and `ParticleLike`.

Reply:
I ended up putting in a link that goes to ParticleLike which describes what's accepted as an input for a particle.